tests/functional/storageclass/test_rbd_default_storageclass.py

```diff
@@ -0,0 +1,72 @@
+import logging
+import pytest
+from ocs_ci.helpers.helpers import is_rbd_default_storage_class
+from ocs_ci.ocs import constants
+from ocs_ci.utility import templating
+from ocs_ci.helpers.helpers import create_unique_resource_name
+from ocs_ci.helpers.helpers import default_storage_class
+from ocs_ci.framework.pytest_customization.marks import (
+    tier1,
+    green_squad,
+    polarion_id,
+    baremetal_deployment_required,
+    ui_deployment_required,
+)
+from ocs_ci.ocs.ui.helpers_ui import is_ui_deployment
+
+
+log = logging.getLogger(__name__)
+
+
+@green_squad
+@baremetal_deployment_required
+@ui_deployment_required
+class TestRBDStorageClassAsDefaultStorageClass:
+    @tier1
+    @polarion_id("OCS-5459")
+    def test_pvc_creation_without_storageclass_name(self, pvc_factory, pod_factory):
+        """
+        Test PVC creation without mentioning storageclass name in the spec.
+
+        Steps:
+            1. Verify RBD storageclass  is set as default.
+            2. Create a PVC and don't provide any storage class name in the YAML file.
+            3. Verify PVC has created and it has attached to the Default RBD  SC.
+            4. Create a POD and attached the  above PVC to the Pod.
+            5. Start IO on verify that IO is successful on the PV.
+        """
+        if not is_ui_deployment():
+            pytest.skip("cluster is not deployed from UI. Skipping test.")
```

Comment:
`ui_deployment_required is an early quick check to verify the UI deployment via config, and is_ui_deployment thoroughly checks the UI deployment with multiple checks. Since we can't use the latter in the marker, I have put it into the test.